### DISCUSSION TITLE
I've downgraded firmware, I can see my Tapo C210 stream in HomeAssistant but can't see it in HomeKit

### DISCUSSION THREAD

**Discussion by YeboMate** [JurajNyiri/HomeAssistant-Tapo-Control#686] [Q&A]
Hi, I seem to have got it working by downgrading the firmware to a supported version. Then got it all setup and can see the camera in Home Assistant but I can't see to get the video stream to show up in HomeKit. 
Any help would be greatly appreciated. 
Thank you.

**Comment by ianmcginley** [comment]
Same, just got my new C510W's, can see them in Home-Assistant, and I have Camera's included in the Home-Assistant HomeKit Bridge - restarts of home-assistant doesn't bring it through like it does for most things with HA + HK.

**Comment by Skyman81** [comment]
I have a similar problem with a TAPO C400… I see everything on HA but on HK only the screenshots and not the streaming
